src/oneapi/oneapi_backend.h

```diff
@@ -40,11 +35,11 @@ class _ONEAPI_BACKEND_EXPORT PySyclExecutionContext
     void apply();
 
 private:
-    daal::services::SyclExecutionContext * m_ctxt;
+    daal::services::internal::ExecutionContext * m_ctxt;
```

Comment:
Why you changed this?

Reply:
So as not to include the sycl daal in this file. This made it possible to remove the DCPP compiler for compiler `oneapi.pyx`